geniza/corpus/migrations/0020_populate_documenttype_display_label.py

```diff
@@ -36,5 +36,5 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunPython(populate_document_types),
+        migrations.RunPython(populate_document_types, migrations.RunPython.noop),
```

Comment:
Oh! I didn't know about this. This is super helpful for another PR I'm working on.